### PR TITLE
Add stub file for IDE exploration

### PR DESCRIPTION
**Pull request by matthieu-rolland**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Following autoload refactor, some navigation feature were unavailable in IDEs such as PHPSTORM
| Type?             | improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | This PR has zero impact on the behavior of prestashop as a software.
| Fixed ticket?     | none
| Related PRs       | 
| Sponsor company   | PrestaShop SA

### CODE REVIEW

**Review by FabienPapet — DISMISSED** [review]

**Review by zuk3975 — DISMISSED** [review]

**Review by PululuK — COMMENTED** [review]
Is it possible to order this list by class names? Easier to detect if a class is missing

**Review by jolelievre — DISMISSED** [review]
OK for me but the suggestion from @PululuK is relevant since it's manually added it can be sorted alphabetically by class name to find missing classes more easily (also will help for future updates). Not blocking though

File: .gitignore [hunk 1]
```diff
@@ -200,6 +200,7 @@ themes/default-bootstrap/modules/*/mails/*
 /var/sessions/*
 !var/sessions/.gitkeep
 !var/SymfonyRequirements.php
+!var/class_stub.php.php
```
Comment (zuk3975) [inline .gitignore]:
```suggestion
!var/class_stub.php
```

**Review by zuk3975 — DISMISSED** [review]

File: var/class_stub.php [hunk 2]
```diff
@@ -0,0 +1,271 @@
+
+
+
```
Comment (jolelievre) [inline var/class_stub.php]:
```suggestion
```

**Review by jolelievre — APPROVED** [review]
Thanks @matthieu-rolland

**Review by mflasquin — APPROVED** [review]

**Review by zuk3975 — APPROVED** [review]

**Comment by eternoendless** [comment]
This isn't created automatically anymore?

**Comment by PrestaEdit** [comment]
> This isn't created automatically anymore?

Since #29923 , it's no more the case.

**Comment by nicosomb** [comment]
No need to have a QA review and no need to launch ga.tests.ui.pr tool.

**Review by nicosomb — APPROVED** [review]

**Review by FabienPapet — APPROVED** [review]

**Comment by FabienPapet** [comment]
Thank you @matthieu-rolland !

**Pull request merged**